packages/api/src/auth/decoders/index.ts

```diff
@@ -31,6 +31,7 @@ const typesToDecoders: Record<
   firebase: noop,
   supabase: supabase,
   ethereum: ethereum,
+  nhost: noop,
```

Comment:
Just double checking that there is no way to verify the token?

Reply:
@Tobbe In production not really, the JWT token is issued by HBP on the fly (our auth solution) and then verified and decoded by the Hasura GraphQL engine.

Reply:
Alright, at least we've thought about it an extra round 👍

Reply:
Would an nhost user use our api side? Maybe we should throw and say something like... use nhost instead?

Reply:
> @Tobbe In production not really, the JWT token is issued by HBP on the fly (our auth solution) and then verified and decoded by the Hasura GraphQL engine.

Here I meant "in development"